packages/client/src/StreamrClient.ts

```diff
@@ -102,13 +102,13 @@ export class StreamrClient {
      * @category Important
      *
      * @param streamDefinition - the stream or stream partition to publish the message to
-     * @param content - the content (the payload) of the message
+     * @param content - the content (the payload) of the message (usually an JSON serializable object)
```

Comment:
Needs to be JSON serializable object as we create a message which has content type of JSON (the default content type, `StreamMessage:107`)

Reply:
Addressed in a5e9c88e854ce597f323272888a637a5150aa834 with wording change "must be"